contrib/google.golang.org/grpc/stats_client.go

```diff
@@ -0,0 +1,60 @@
+package grpc
+
+import (
+	"net"
+
+	context "golang.org/x/net/context"
+	"google.golang.org/grpc/stats"
```

Comment:
Can we please have a blank line between the tracer group of imports and the rest? Just like we do for the standard library.